README.md

```
@@ -71,14 +71,19 @@ Dela includes an [MCP (Model Context Protocol)](https://modelcontextprotocol.io/
 
 The mcp executed tasks need to be already on the allowlist. The mcp server respects the allowlist, but does not give agents tools to modify it. 
 
-### Setting Up MCP in Your Editor
+### Setting Up MCP for your agent
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)